types/office-js-preview/index.d.ts

```diff
@@ -10785,7 +10786,8 @@ declare namespace Office {
          *
          * **Note**: The following clients have different behavior for saveAsync on appointments in compose mode:
          *
-         * - Mac Outlook does not support saveAsync on a meeting in compose mode. Calling saveAsync on a meeting in Mac Outlook will return an error.
+         * - Outlook for Mac does not support saveAsync on a meeting in compose mode. As such, calling saveAsync in that scenario returns an error.
```

Comment:
I recommend using this text from another PR you created. I like this wording better. I think it reads easier.

Outlook for Mac does not support saving a meeting. The `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.

Reply:
I'd recommend that text throughout all of these files.